src/Components/Users/Users.tsx

```diff
@@ -1,7 +1,172 @@
-import React from 'react'
+import { Fragment, useContext, useEffect, useState } from 'react'
+import styles from './Users.module.css'
+import { HiDotsVertical } from "react-icons/hi";
+import { FaEye } from "react-icons/fa6"
+import { ImBlocked } from "react-icons/im"
+import { CiSearch } from "react-icons/ci";
+import { IoFilter } from "react-icons/io5";
+import { AuthContext } from '../../Context/AuthContext';
+import axios from 'axios';
+import { Modal } from 'react-bootstrap';
 
 export default function Users() {
+
+  const { baseUrl, reqHeaders }: any = useContext(AuthContext)
+  const [usersList, setUsersList] = useState()
+  const [modalState, setModalState] = useState("close")
+  const handleClose = () => setModalState("close");
+  const [userItem, setUserItem] = useState()
+
+  const showAllUsers = () => {
+    axios.get(`${baseUrl}/Users`, { headers: reqHeaders }).then((response) => {
+      setUsersList(response.data.data);
+    }).catch((error) => {
+      console.log(error);
```

Comment:
logs should be removed or commented  before merging into the production branch to avoid revealing sensitive data to users